src/collect/cli.rs

```diff
@@ -58,6 +58,13 @@ Example: --probe tp:skb:kfree_skb --probe kprobe:consume_skb"
         default_value = "false"
     )]
     pub(super) print: bool,
+    #[arg(
+        long,
+        help = "Pretty-print the event in a given format",
+        value_parser=PossibleValuesParser::new(["singleline", "multilines"]),
```

Comment:
s/multilines/multiline?

Reply:
Also, if you use the enum directly as the field type clap will generate the possible values for you (con: you need the num to derive from clap::ValueEnum)